apps/demo-app/package.json

```
@@ -18,14 +18,17 @@
     "@nuxtjs/color-mode": "^3.5.2",
     "@nuxtjs/i18n": "^10.0.6",
     "@pinia/nuxt": "~0.11.2",
+    "@sit-onyx/chartjs-plugin": "1.0.0-beta.299",
```

```suggestion
    "@sit-onyx/chartjs-plugin": "workspace:^",
```

You also need to run pnpm instsall afterwards to update the lock file